### DISCUSSION TITLE
GCP Secrets Manager

### DISCUSSION THREAD

**Discussion by luis-n4489** [concourse/concourse#7283] [Help & Support]
I have installed Concourse many times w/ Vault as a source for secrets + AWS deployments. Now that I am moving onto a GCP deployment of concourse (Helm chart + k8s) i need to pull secrets from GCP and i swear i thought i saw documentation in regards to setting up - but for the life of me i cannot find it anymore. Can someone confirm I can or cannot use gcp secrets manager please?

**Comment by taylorsilva** [comment]
Currently no support for GCP secrets. If there's a way to mirror them to k8s secrets then that could be a way to hack around it.

**Comment by luis-n4489** [comment]
But i can use k8's secrets?

**Reply by taylorsilva** [reply]
Yes! k8s secrets are supported

**Comment by luis-n4489** [comment]
Sorry to bring this back from the grave, I have searched but cannot find
any information on how to configure the values +k8s secrets using the helm
chart. any link is appreciated

**Reply by taylorsilva** [reply]
Did you see this page? https://concourse-ci.org/kubernetes-credential-manager.html
Don't think we wrote anything up specific to the helm chart.
All the config values for k8s secret in the chart are here: https://github.com/concourse/concourse-chart/blob/48f5c21abddc6b35f46731bd66c1ae988181c528/values.yaml#L426-L455

**Comment by luis-n4489** [comment]
yea! sorry should've replied back sooner. I successfully ran a pipeline w/
a secret echo'ed out . thanks!
